assets/sass/components/_navbar.scss

```diff
@@ -0,0 +1,128 @@
+$navbar-background-color: $brand-primary !default;
+$navbar-text-color: choose-contrast-color($brand-primary, white, black) !default;
+
+/* site header */
+.topbar {
+  background-color: $navbar-background-color;
+  border-bottom: none;
+  -webkit-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25), inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+  -moz-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25), inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25), inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+}
+.topbar .navbar-brand,
+.topbar .navbar-nav>li>a,
+.topbar .navbar-toggle,
+.topbar .navbar-nav>.active>a,
+.topbar .navbar-nav>.active>a:focus,
+.topbar .navbar-nav>.active>a:hover {
+  color: $navbar-text-color;
+  background: transparent;
+}
+.topbar a.navbar-brand:hover,
+.topbar .navbar-nav>li>a:hover,
+.topbar .navbar-nav>.open>a,
+.topbar .navbar-nav>.open>a:focus,
+.topbar .navbar-nav>.open>a:hover,
+.navbar-inverse .navbar-toggle:focus,
+.navbar-inverse .navbar-toggle:hover {
+  background-color: darken($navbar-background-color, 10%);
+}
+.topbar .navbar-toggle .badge {
+  position: absolute;
+  right: -10px;
+  top: 4px;
+}
+.topbar .navbar-header .visible-xs {
+  max-width: 250px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+.topbar ul.visible-xs {
+  margin: 0;
+}
+.topbar ul.visible-xs li {
+  border: 1px solid;
+  border-top: none;
+  border-color: darken($navbar-background-color, 25%);
+}
+.topbar ul.visible-xs li a {
+  color: $navbar-text-color !important;
+  padding: 10px 15px !important;
+}
+.topbar ul.visible-xs ul.dropdown-menu {
+  padding: 0;
+}
+.topbar ul.visible-xs ul.dropdown-menu li {
+  border: none;
+  border-top: 1px solid;
+}
+.topbar ul.visible-xs .header {
+  border-left: none;
+  border-right: none;
+  padding: 0.2em 1em;
+}
+.topbar ul.visible-xs .header h4 {
+  font-size: 10px;
+  font-weight: bolder;
+  color: #75A5D0;
+  margin: 0;
+}
+.navbar-nav>li.menu-groups>a.dropdown-toggle {
```

Comment:
Should these use nested SASS format?

Reply:
I didn't re-write the whole file yet, only added the variables. I'll re-write it before this gets merged.

Reply:
Maybe I can just remove this file for now. There are bigger problems with the navbar which need to be addressed, so I'm afraid that a lot of these styles may become redundant.